src/GovUk.Education.ExploreEducationStatistics.Data.Services/AbstractSubjectMetaService.cs

```diff
@@ -119,7 +120,7 @@ protected static IEnumerable<IndicatorMetaViewModel> BuildIndicatorViewModels(IE
             {
                 Label = filterGroup.Label,
                 Options = filterItems
-                    .OrderBy(item => item.Label, LabelComparer)
+                    .OrderBy(item => item.Label.ToLower() != "total").ThenBy(item => item.Label, LabelComparer)
```

Comment:
Maybe just bang a newline before the `.ThenBy` just to keep the formatting consistent with the first occurrance?